introduction/java_int_to_string/Solution.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+import java.security.*; //dont know what exactly is used, cant check properly
+
+public class Solution {
+  public static void main(String[] args) {
+    DoNotTerminate.forbidExit();
+    try {
+      Scanner input = new Scanner(System.in);
+      int inputNumber = input.nextInt();
+      input.close();
+      String stringFromNumber = new String(Integer.toString(inputNumber));
```

Comment:
`String stringFromNumber = Integer.toString(inputNumber);`

No need for extra copy constructor.